Frontend/ui-customer/spago.dhall

```diff
@@ -30,6 +30,7 @@ to generate this file without the comments in this block.
   , "functions"
   , "halogen-vdom"
   , "integers"
+  , "lists"
```

Comment:
@MercyQueen Why is this required, I don't see any usage in this PR

Reply:
@kavya-shree-s its not being used but while generating bundle, we are getting error, hence I've added it here.